Makefile

```diff
@@ -1,17 +1,28 @@
+SHELL=/bin/bash
+
 CFLAGS= -Wall -Wextra
 CFLAGS+= -O0 -ggdb
 CFLAGS+= -std=c99
 
 LDFLAGS= -lxcb-randr -lxcb
 
-all: xrandr-invert-colors.bin
+PREFIX=/usr/local/bin
```

Comment:
Maybe PREFIX should be only `/usr/local`, and the `bin` part should be added separately, because as far as I jave seen this is the (more) standard:  https://www.gnu.org/prep/standards/html_node/Directory-Variables.html
Feel free to correct me...